### PR TITLE
openssh: update gssapi patch, fix the build

### PR DESCRIPTION
**Pull request by benley**
Backporting this build fix from master.

### CODE REVIEW

File: pkgs/tools/networking/openssh/default.nix [hunk 1]
```diff
@@ -17,9 +17,10 @@ let
     sha256 = "682b4a6880d224ee0b7447241b684330b731018585f1ba519f46660c10d63950";
   };
 
+  # **please** update this patch when you update to a new openssh release.
```
Comment (rasendubi) [inline pkgs/tools/networking/openssh/default.nix]: How would I update the patch?

Comment (benley) [inline pkgs/tools/networking/openssh/default.nix]: unfortunately the only way I know of is to find a newer version in the Debian source repo ... which may not be available right away when there is an openssh release :-/

**Comment by benley** [comment]
OK to merge?  In theory this can't break anything, since it's already broken...

**Comment by rasendubi** [comment]
Is that fixed in master?

**Comment by benley** [comment]
Yes - this was a cherrypick from master.

**Pull request merged**